benchmark.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+which php &> /dev/null
+if [ $? != 0 ];
+then
+	echo 'cannot find php';
+	exit 255;
+fi
+
+which grep &> /dev/null
+if [ $? != 0 ];
+then
+	echo 'cannot find grep';
+	exit 255;
+fi
+
+which awk &> /dev/null
+if [ $? != 0 ];
+then
+	echo 'cannot find awk';
+	exit 255;
+fi
+
+if [ "" == "$1" ] || [ "" == "$2" ];
+then
+	echo "Usage: bash benchmark.sh BRANCH1 BRANCH2 ...BRANCHN"
+	exit 1;
+fi
+
+for BRANCH in $@
+do
+	git checkout $BRANCH &> /dev/null &&
+	git reset --hard &> /dev/null &&
+	echo -n $BRANCH
+	(for i in `seq 1 10`; do php php-cs-fixer fix . ; done) | grep -i fixed | awk '
```

Comment:
next one... I don't have access to that `seq`. May you replace it with `{1..10}` ?